packages/wallets/src/components/SentEmailContent/SentEmailContent.tsx

```diff
@@ -2,22 +2,63 @@ import React, { FC, Fragment, useEffect, useState } from 'react';
 import { Trans } from 'react-i18next';
 import { useCountdown } from 'usehooks-ts';
 import { useActiveWalletAccount, useSettings, useVerifyEmail } from '@deriv/api-v2';
-import { DerivLightIcEmailSentIcon } from '@deriv/quill-icons';
+import {
+    DerivLightIcEmailSentIcon,
+    DerivLightIcFirewallEmailPasskeyIcon,
+    DerivLightIcSpamEmailPasskeyIcon,
+    DerivLightIcTypoEmailPasskeyIcon,
+    DerivLightIcWrongEmailPasskeyIcon,
+} from '@deriv/quill-icons';
 import { PlatformDetails } from '../../features/cfd/constants';
 import useDevice from '../../hooks/useDevice';
+import ChangePasswordIcon from '../../public/images/change-password-email.svg';
```

Comment:
Let's use icons from `deriv/quill-icon` instead here is the icon `DerivLightIcEmailSentPasskeyIcon`

Reply:
Ooo they have that, I missed it, tenks 👀